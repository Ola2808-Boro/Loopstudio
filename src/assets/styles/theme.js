export const theme = {
  colors: {
    white: "#ffffff",
    black: "	#000000",
    red: "#FF0000",
    darkGray: "	#8c8c8c",
    veryDarkGray: "#696969",
    vividMulberry: "#C505EB",
    lavenderIndigo: "#8647F8",
  },
  fontSizes: {
    s: "15px",
    m: "20px",
    l: "25px",
    xl: "30px",
  },
};
