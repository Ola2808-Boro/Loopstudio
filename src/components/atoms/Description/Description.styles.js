import styled from "styled-components";
import { device } from "helpers/breakpoints";

export const StyledDescription = styled.p`
  color: ${({ theme }) => theme.colors.darkGray};
  font-size: ${({ theme }) => theme.fontSizes.m};
  font-weight: bold;
  font-family: "Alata", sans-serif;
  text-align: center;

  @media ${device.desktop} {
    text-align: left;
  }
`;
