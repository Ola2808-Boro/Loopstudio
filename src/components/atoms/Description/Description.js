import React from "react";
import PropTypes from "prop-types";
import { StyledDescription } from "components/atoms/Description/Description.styles";
const Description = ({ description }) => {
  return <StyledDescription>{description}</StyledDescription>;
};

Description.propTypes = {
  description: PropTypes.string.isRequired,
};

export default Description;
