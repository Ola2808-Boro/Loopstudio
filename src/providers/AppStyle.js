import React from "react";
import { ThemeProvider } from "styled-components";
import { theme } from "assets/styles/theme";
import GlobalStyle from "assets/styles/GlobalStyle";
import "assets/styles/fonts.css";

const AppStyle = ({ children }) => {
  return (
    <ThemeProvider theme={theme}>
      <GlobalStyle />
      {children}
    </ThemeProvider>
  );
};

export default AppStyle;
