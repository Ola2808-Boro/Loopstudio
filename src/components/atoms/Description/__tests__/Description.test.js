import "jest-styled-components";
import renderer from "react-test-renderer";
import { render, screen } from "@testing-library/react";
import AppStyle from "providers/AppStyle";
import { StyledDescription } from "components/atoms/Description/Description.styles";
describe("Test button component", () => {
  it("default style", () => {
    const tree = renderer
      .create(
        <AppStyle>
          <StyledDescription />
        </AppStyle>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
