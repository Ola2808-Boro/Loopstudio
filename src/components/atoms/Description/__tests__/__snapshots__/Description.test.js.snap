// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test button component default style 1`] = `
<p
  className="eobklK"
/>
`;
